test/controllers/comments_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class CommentsControllerTest < ActionController::TestCase
+  # test "the truth" do
```

Comment:
I am so far from being done with my tests, I am sorry if I gave the impression they were completed.  It was my goal today to try and really shore them up.